python/cudf/cudf/core/buffer.py

```diff
@@ -60,10 +60,19 @@ def __init__(self, data=None, size=None, owner=None):
     def __reduce__(self):
         return self.__class__, (self.to_host_array(),)
 
+    @property
+    def __cuda_array_interface__(self):
+        intf = {
+            "data": (self.ptr, False),
+            "shape": (self.size,),
+            "strides": (1,),
+            "typestr": "|u1",
+            "version": 0,
```

Comment:
Are versions backwards compatible? Afraid of someone saying "hey we need version >= 2"

Reply:
Yeah this should be consistent with newer versions. FWIW we've been doing the same in `DeviceBuffer` as well.